string id
string ref
int16 floor_number
geometry_msgs/Pose waypoint_pose